controllers/components/ApiComponent.php

```diff
@@ -124,32 +124,35 @@ public function search($args)
         $resourceDao = MidasLoader::loadModel("Item")->initDao("Resource", $item->toArray(), "journal");
         $rating = MidasLoader::loadModel("Itemrating", 'ratings')->getAggregateInfo($item);
         $authors = join(", ", $resourceDao->getAuthorsFullNames());
+        // Check that the submission isn't under the "Private" folder of the given issue
+        $parentFolder = $item->getFolders();
+        if($parentFolder[0]->getName() != "Private") {
```

Comment:
It was the one of the easiest ways to differentiate the folder from the others.  The file itself can't be marked as "truly" private since it still needs to be downloaded by anyone who views the review.   

The other option is to examine the metadata tags of the submission to look for the absence of the keys that are set by the regular submission process.